package.json

```diff
@@ -23,6 +23,7 @@
   "devDependencies": {
     "babel-brunch": "~6.0.0",
     "babel-preset-es2015": "^6.6.0",
+    "babel-preset-es2016": "^6.16.0",
```

Comment:
Oh, good thought. I made the assumption that es2015 might still be in use but perhaps that's not the case.